specification/migrate/resource-manager/Microsoft.Migrate/AssessmentProjects/preview/2023-04-01-preview/migrate.json

```diff
@@ -17458,6 +17458,14 @@
           "$ref": "#/definitions/WindowsServerLicensingSettings",
           "description": "Windows Server licensing settings."
         },
+        "rhelLinuxServerLicensing": {
+          "$ref": "#/definitions/LinuxServerLicensingSettings",
```

Comment:
Does it make sense to have linuxServerLicensing as property and osType as one of the properties in it? This will help you accomodate other linux versions too (eg OracleLinux) in future if necessary.

Reply:
Hi @ajaygit158 
This is by design. We dont anticipate any other Linux versions in the current modelling. We have a new modelling in development for BusinessCaseV2 resource that may see growing versions. We will accomodate this suggestion there as that list may increase. 
The current V1 version of the resource will be deprecated after a while once V2 resource is introduced.